src/pages/index.tsx

```diff
@@ -16,6 +16,12 @@ const DEFAULT_ZOOM = 3;
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Reminder: We need to look into typing GraphQL responses. Maybe this markdown plugin has some predefined types for us.